tests/optimizations/run_all.py

```diff
@@ -33,12 +36,8 @@
 sys.path.insert(
     0,
     os.path.normpath(
-        os.path.join(
-            os.path.dirname(os.path.abspath(__file__)),
-            "..",
-            ".."
-        )
-    )
+        os.path.join(os.path.dirname(os.path.abspath(__file__)), "..", "..")
+    ),
 )
```

Comment:
You are a victim here of the commit hook now covering `run_all.py`, but there are no isort:start to make sure the `nuitka` package can be imported. Please add `# isort:start` in much the same way I did in some other runners (can search for it), and it is my fault. I ought to have added these immediately to of all them.